src/ai/ai-bedrock-cohere-embedding-model.ts

```diff
@@ -0,0 +1,15 @@
+export interface CohereEmbeddingBody {
+  texts: string[];
+  input_type:
```

Comment:
any explanation here would be welcome; I saw your comment on the PR, but anyone seeing your code should know as well the use of `isQuery` and why it's important :)

recap, documenting is done like this:
```ts
/** documentation of the interface */ 
interface myInterface {
  /** documentation of the attribute */
  myAttribute: string;
}
```

Reply:
It also doesn't explain why there is classification and clustering

+ 

maybe for clarification I'd replace by a simpler `isQuery: boolean;` making it quite obvious when combined with the explanation


+ 

finally, if you are not using the interface elsewhere, maybe put it in the same document as the function, and don't export it; it reduce noise (I know this is not what we did until now, I'm learning as well)